conformance/lang/expressions/unary-expression/unary_minus_expression.balt

```diff
@@ -507,6 +507,7 @@ function init() {
 
 Test-Case: output
 Description: Test unary negative expression for positive floats with no overflow scenarios.
+Fail-Issue: ballerina-platform/ballerina-spec#1046
```

Comment:
Usually we break such tests into 2 or more tests, where only the incorrect case will have an associated fail issue in a separate test. We will send a separate PR for this.

Reply:
We will also make the tests removed in `boolean_literal.balt` error tests in a separate PR.

Reply:
fixed.